source/FreeRTOS_DHCPv6.c

```diff
@@ -325,7 +325,7 @@ void vDHCPv6Process( BaseType_t xReset,
             }
             else
             {
-                FreeRTOS_printf( ( "vDHCPv6Process: malloc failed %u byt4es\n", sizeof( *pxEndPoint->pxDHCPMessage ) ) );
+                FreeRTOS_printf( ( "vDHCPv6Process: malloc failed %lu byt4es\n", sizeof( *pxEndPoint->pxDHCPMessage ) ) );

```

Comment:
We never use the 'l' prefix in de format string because it has different meanings on different  platforms.

I found that the following gives no warnings on any of the platforms:

~~~
FreeRTOS_printf( ( "vDHCPv6Process: malloc failed %u bytes\n",
    ( unsigned ) sizeof( *pxEndPoint->pxDHCPMessage ) ) 
~~~
It also corrects `byt4es` -> `bytes`.